moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -219,10 +221,9 @@ const ComponentForm = ({
         </Grid>
         <Grid item xs={12}>
           <TextField
-            inputRef={register}
+            {...register("description")}
```

Comment:
Similarly, `register` is no longer assigned as the ref and is invoked to return `ref` and other props that are spread into these components.